requirements.txt

```diff
@@ -1,4 +1,4 @@
-requests==2.25.1
+requests>=2.28.1
```

Comment:
Any reason for `>=` and not locking to a version?

Reply:
Just for better compatibility with other code, where the lib will be used